Common/System/NativeApp.h

```diff
@@ -87,3 +87,6 @@ void NativeShutdownGraphics();
 void NativeShutdown();
 
 void PostLoadConfig();
+
+// The lambda runs on the next update.
+void NativeRunOnMainThread(std::function<void()> func);
```

Comment:
How have we defined main thread, given Vulkan/OpenGL/etc.?  I guess this would be the UI thread (the CPU thread basically) snice it's called from NativeUpdate?

-[Unknown]

Reply:
It's the NativeUpdate thread, basically.

I thought I would use this for something that I didn't end up using it for, so I think I'm just gonna take it out from the PR for now.